packages/pages/lib/css/layouts/wrapper.css

```
@@ -0,0 +1,20 @@
+.wrapper {
+	box-sizing: border-box;
+	padding-left: 1.25rem;
+	padding-right: 1.25rem;
+	width: 100%;
+}
+
+@media (min-width: 768px) {
+	.wrapper {
+		padding-left: 3rem;
+		padding-right: 3rem;
+	}
+}
+
+@media (min-width: 1280) {
```

You silly silly goose 🦆

```suggestion
@media (min-width: 1280px) {
```